.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: ruby
+rvm:
+- 2.3.0
+before_script:
+- bundle exec rake db:create --all
+- bundle exec rake db:migrate
+script:
+- bundle exec rake ci:tests
+services:
+- postgresql
+# deploy:
+  # provider: heroku
+  # app: garden-club
```

Comment:
Then you should add it at that point. Let's try to keep the code as clean as possible to make the review process simple.